docker-compose.yml

```diff
@@ -18,8 +18,8 @@ services:
       retries: 5
     command: ['postgres', '-c', 'fsync=off', '-c', 'synchronous_commit=off', '-c', 'full_page_writes=off']
   redis:
-    image: balena/balena-redis:v0.0.4
-    command: [sh, -c, "redis-server /usr/local/etc/redis/redis.conf --save ''"]
+    image: redis:6-alpine
+    command: [sh, -c, "redis-server --save '' --always-show-logo yes"]
```

Comment:
@ramirogm 
- Might get better performance with the standard `redis:6` image instead of the Alpine variant.
- Why `--always-show-logo yes`?